src/Core/Domain/Product/QueryResult/ProductForEditing.php

```diff
@@ -281,4 +176,9 @@ public function getCoverThumbnailUrl(): string
     {
         return $this->coverThumbnailUrl;
     }
+
+    public function getDateNew(): DateTime
```

Comment:
As suggested above, this new field should be added in the `ProductOptions` sub DTO